blockdevice_test.go

```diff
@@ -41,6 +42,11 @@ func (*blockdeviceSuite) TestReadBlockDevices(c *gc.C) {
 	partition := partitions[0]
 	c.Check(partition.ID(), gc.Equals, 1)
 	c.Check(partition.UsedFor(), gc.Equals, "ext4 formatted filesystem mounted at /")
+
+	fs := blockdevice.FileSystem()
+	c.Assert(fs, gc.NotNil)
```

Comment:
Can you add a check for the actual content

Reply:
The `Type()` and `MountPoint()` are checked below? Is there something different your looking for?

Reply:
Yeah, what's below does the job